File: project/RocatMonitorVisualizer/Assets/Scripts/Message/Message.cs
using MsgPack.Serialization;
using System.Collections.Generic;

public class Message
{

  [MessagePackMember(0)]
  public long CurrentTime = 0;

  [MessagePackMember(1)]
  public double TimeLength = 0;

  [MessagePackMember(2)]
  public List<MethodInfo> Methods = null;

  [MessagePackMember(3)]
  public List<ExeTimeInfo> ExeTimes = null;

}

File: project/Heijoroid/app/src/main/java/jp/naist/heijo/android/message/Message.java
package jp.naist.heijo.android.message;

import java.util.LinkedList;
import java.util.List;

@org.msgpack.annotation.Message
public class Message
{

  @org.msgpack.annotation.Index(0)
  public long CurrentTime = 0;

  @org.msgpack.annotation.Index(1)
  public double TimeLength = 0;

  @org.msgpack.annotation.Index(2)
  public List<MethodInfo> Methods = new LinkedList<>();

  @org.msgpack.annotation.Index(3)
  public List<ExeTimeInfo> ExeTimes = new LinkedList<>();

}
